app/views/users/index.html.erb

```diff
@@ -1 +1,7 @@
-<h1>Users here</h1>
\ No newline at end of file
+<h1>Users here</h1>
+
+<% @users.each do |user| %>
+
+    <%= render 'shared/users', user: user %>
+    
+<% end %>
```

Comment:
- [ ] kindly match the given wireframe 
your design :point_down: 
![Screenshot from 2023-07-20 22-18-20](https://github.com/joyapisi/blog_app/assets/66097371/ecc07b3d-1595-4f3c-a13f-2dc7f06f01e4)
wireframe :point_down: 
![Screenshot from 2023-07-20 22-19-45](https://github.com/joyapisi/blog_app/assets/66097371/bff15849-c287-4c13-ae10-a0b3080f6ca9)